smart-contracts/contracts/mixins/MixinKeys.sol

```diff
@@ -272,8 +272,8 @@ contract MixinKeys is
   ) internal view
     returns (bool)
   {
-    if(keyManagerOf[_tokenId] == msg.sender ||
```

Comment:
Should there be a pair of tests which make sure it passes with `_keyManager` and fails with `msg.sender` in the case where the sender is not the `_keyManager`?

Reply:
I could add this. This function is not tested directly yet.

Reply:
added some tests.